src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -634,7 +634,8 @@ private String upper(String property, String culture) {
         }
 
         if (mathConstants.contains(functionName) || numberOperators.contains(functionName)
-                || trigFunctions.contains(functionName) || binaryOperators.contains(functionName)) {
+                || trigFunctions.contains(functionName) || binaryOperators.contains(functionName)
+                || utilityFunctions.contains(functionName)) {
```

Comment:
a minor concern: there is another `field` function (and probably more other functions added in future) in `utilityFunctions`. We may need to have a mapping for result type of specific function later.

Reply:
@dai-chen  Yeah, it is. So do you think we need to leave out the `field` and put `functionNama.equals("assign") in the condition clause instead?

Reply:
It's okay. We can do this later because I assume you will touch this part of code soon. :)

Reply:
Ok, thanks!